lib/plugins/aws/invokeLocal/index.test.js

```diff
@@ -503,6 +503,13 @@ describe('AwsInvokeLocal', () => {
 
       expect(serverless.cli.consoleLog.lastCall.args[0]).to.contain('"errorMessage": "failed"');
     });
+
+    it('should throw when module loading error', () => {
+      awsInvokeLocal.serverless.config.servicePath = __dirname;
+
+      expect(() => awsInvokeLocal.invokeLocalNodeJs('fixture/handlerWithLoadingError',
+        'anyMethod')).to.throw(Error);
```

Comment:
We could check for the correct message here instead of a generic error, to be sure that it is catched as expected:
`.to.throw(/Exception encountered when loading/);`

Reply:
Done!